packages/marshal/src/passStyleOf.js

```diff
@@ -198,35 +225,59 @@ function isPassByCopyRecord(val) {
  *
  * @param {{ [PASS_STYLE]: string, [Symbol.toStringTag]: string, toString: () =>
  * void}} val the value to verify
```

Comment:
Is doing this TODO easy? In any case, whitespace could be better, which prettier won't help or hurt with since it is in a comment

```suggestion
 *
 * @param {{ [PASS_STYLE]: string,
 *           [Symbol.toStringTag]: string,
 *           toString: () => void
 * }} val the value to verify
```

Reply:
I can't do the todo.  I have at least formatted it better.